### PR TITLE
 Bootstrap with Laravel #6 

### PR DESCRIPTION
**Pull request by raphaeldealmeida**
Bootstrap with Laravel #6 
It needs volume log review

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,13 @@
...
+/storage/*.key
+/vendor
+.env
+.env.backup
+.phpunit.result.cache
+Homestead.json
+Homestead.yaml
+npm-debug.log
+yarn-error.log
+.idea
```
Comment (vitormattos) [inline .gitignore]: `.docker/volumes/mysql`
Add this folder to .gitignore


`.docker/volumes/storage`
Add this folder to gitignore

File: README.md [hunk 2]
```diff
@@ -13,6 +13,15 @@ Users should log in to the system and view invoices and recording audios of all
 
 The project needs to be internationalized. Do not display any text to the user that cannot be translated.
 
+## Instructions to getting run
```
Comment (vitormattos) [inline README.md]: Good! :+1:

File: app/Http/Controllers/Auth/LoginController.php [hunk 3]
```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Http\Controllers\Auth;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Foundation\Auth\AuthenticatesUsers;
+
+class LoginController extends Controller
```
Comment (vitormattos) [inline app/Http/Controllers/Auth/LoginController.php]: The login is working fine in this PR?

File: database/migrations/2014_10_12_000000_create_users_table.php [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+class CreateUsersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('users', function (Blueprint $table) {
```
Comment (vitormattos) [inline database/migrations/2014_10_12_000000_create_users_table.php]: Good! :+1:

File: database/seeds/DatabaseSeeder.php [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+class DatabaseSeeder extends Seeder
+{
+    /**
+     * Seed the application's database.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        // $this->call(UsersTableSeeder::class);
```
Comment (vitormattos) [inline database/seeds/DatabaseSeeder.php]: Don't is necessary put the admin user?

File: storage/logs/laravel-2019-10-17.log [hunk 6]
```diff
@@ -0,0 +1,15 @@
+[2019-10-17 09:33:28] local.DEBUG: TEste  
+[2019-10-17 09:33:29] local.DEBUG: TEste
```
Comment (vitormattos) [inline storage/logs/laravel-2019-10-17.log]: Put .gitignore into log folder

File: database/seeds/DatabaseSeeder.php (hunk 5 above)
Comment (raphaeldealmeida) [reply database/seeds/DatabaseSeeder.php]: It'll be done later

File: app/Http/Controllers/Auth/LoginController.php (hunk 3 above)
Comment (raphaeldealmeida) [reply app/Http/Controllers/Auth/LoginController.php]: No, this is a scaffoling

**Comment by raphaeldealmeida** [comment]
FIX volumes
ignore log files

**Pull request merged**